packages/cli/src/seeder-repl

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env node
+
+const { SDK } = require('@geut/permanent-services')
+
+;(async () => {
```

Comment:
I think we should a `catch` clause. This will help if something would wrong otherwise we could miss it.
Something like this: 
```js
const repl = async () => { ... }

repl().catch(err => { 
  console.error(err) 
  process.exit(1)
}
```